src/main/java/com/epam/spring/homework1/pet/Spider.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.spring.homework1.pet;
+
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Component
+@Order
```

Comment:
It's unnecessary to keep annotation `Order `here. What was important - is to have `Dog `the higher priority, remove it also from `Cheetah`